03/app.js

```diff
@@ -1,18 +1,60 @@
-const min = 1;
-const max = 100;
-
-const a = randomNumber(min, max);
-const b = randomNumber(min, max);
-const c = randomNumber(min, max);
-
-console.log(a, b, c);
-
-
-
-
+const min = 1
+const max = 100
 
+const a = randomNumber(min, max)
+const b = randomNumber(min, max)
+const c = randomNumber(min, max)
 
+console.log(a, b, c)
 
 function randomNumber(min, max) {
-    return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+	return Math.round(Math.random() * (max - min) + min)
+}
+
+function getSum(a, b, c) {
+	if (a > b && c > b) {
+		return a + c
+	}
+	if (b > a && c > a) {
+		return b + c
+	}
+	if (a > c && b > c) {
```

Comment:
Jeśli został by wiżety pod uwagę komentarz poniżej to w zasadzie tutaj już nie trzeba robić warunku.
Skoro dwa poprzednie były nieprawdziwe to zostaje nam tylko to, że a oraz b jest najwięszke  czyli `return a + b` bez `if` ;)